### PR TITLE
General ecological

### PR DESCRIPTION
**Pull request by lokal-profil**
Add a P4002 claim

Also adds another custom Error.

Task: [T166675](https://phabricator.wikimedia.org/T166675)

### CODE REVIEW

File: WFD/mappings.json [hunk 1]
```diff
@@ -59,6 +59,14 @@
...
         "TEMP": "Q2914309"
     },
+    "swEcologicalStatusOrPotentialValue": {
+        "@meta": "https://www.wikidata.org/wiki/User:AndreCostaWMSE-bot/WFD/swEcologicalStatusOrPotentialValue",
+        "1": "Q30092114",
+        "2": "Q30092128",
+        "3": "Q30092136",
+        "4": "Q30092163",
+        "5": "Q30092172"
+    }
```
Comment (Vesihiisi) [inline WFD/mappings.json]: Expected comma here. Also now that I've tried validating the json, there's a trailing comma on line 13 (after last element of array) but apparently I can't leave a comment there.

File: WFD/swb_import.py [hunk 2]
```diff
@@ -213,7 +216,9 @@ def make_significant_impact_type(self, data):
                * In both cases the reference can still be added.
 
         :param data: dict of data for a single swb
+        :return: [Statement]
         """
+        # @todo: Support for UNKN, OTHE (somevalue) + NOTA, NOSI (novalue)
```
Comment (Vesihiisi) [inline WFD/swb_import.py]: If I understand the docstring correctly, if there's no impact type in the data, it means it's been _determined_ there are none? Just trying to wrap my head around the use of 'novalue' here.

File: WFD/mappings.json (hunk 1 above)
Comment (lokal-profil) [inline WFD/mappings.json]: Thanks for spotting this. I switched editor and the new one did not have built in json linting.

Fixed

File: WFD/swb_import.py (hunk 2 above)
Comment (lokal-profil) [inline WFD/swb_import.py]: `novalue` would be used when it has been _decided_ that there are none.

The difference (also for `somevalue`) is between:
* No entry in Wikidata: Cannot say if it has been missed, it is none or it is unknown.
* `novalue` in Wikidata: The official data says none (for this year)
* `somevalue` in Wikidata: The official data says we are not sure (for this year)

Comment (Vesihiisi) [inline WFD/swb_import.py]: :+1:

Comment (Vesihiisi) [inline WFD/swb_import.py]: Is this documented somewhere on Wikidata?

Comment (lokal-profil) [inline WFD/swb_import.py]: Added ([diff](https://www.wikidata.org/w/index.php?title=Property_talk%3AP4002&action=historysubmit&type=revision&diff=496868264&oldid=492845261))

**Pull request merged**